recipes/lora_finetune_distributed.py

```diff
@@ -546,6 +548,20 @@ def _setup_model(
         # synchronize before training begins
         torch.distributed.barrier()
 
+        # Calculate model FLOPs for MFU tracking
+        if self._is_rank_zero:
+            # Calculate model FLOPs for MFU tracking
+            self._model_flops = mfu.get_transformer_flops(
+                hidden_size=cfg_model.hidden_size,
+                intermediate_size=cfg_model.hidden_size * 4,  # Standard transformer uses 4x
+                num_layers=cfg_model.num_layers,
+                seq_len=cfg_model.seq_length
```

Comment:
i this the batch sequence length? The cfg.model.seq_length is just to adjust the RoPE scaling